setup.py

```diff
@@ -144,6 +144,8 @@ def main():
                 'telethon_generator', 'telethon_tests', 'run_tests.py',
                 'try_telethon.py'
             ]),
+            test_suite='run_tests.test_suite',
```

Comment:
Please don't. This is a terrible misfeature of setuptools that isn't necessary with tox.

Reply:
You mean we should run our own test runner? Why?

Reply:
@habnabit I fixed it but still wonder why. Or do you mean not setting `test_suite` at all?